style.css

```diff
@@ -265,7 +265,7 @@ button {
      }
  }
  
- @media screen and (max-width: 550px) {
+ @media screen and (max-width: 550px), screen and (max-width: 650px) and (orientation: landscape) {
      body {
```

Comment:
It still looks bad.
![image](https://user-images.githubusercontent.com/90243506/135692487-f7420303-db1a-4e82-8128-007af23487c2.png)